src/main/java/exams/App.java

```diff
@@ -0,0 +1,26 @@
+package exams;
+
+public class App {
+    public static void main(String[] args) {
+        Student[] listStudents = new Student[5];
+        for (int num_students = 0; num_students < listStudents.length; num_students++)
+            listStudents[num_students] = new Student();
+        int[] results = new int[5];
```

Comment:
you don't need results array - you can calculate sum of marks right in cycle for